krun/vm_defs.py

```diff
@@ -526,7 +526,7 @@ def __init__(self, vm_path, java_path, env=None):
         JRubyVMDef.__init__(self, vm_path, env=env)
         self.add_env_change(EnvChangeAppend("JAVACMD", java_path))
 
-        self.extra_vm_args += ['-X+T', '-J-server']
+        self.extra_vm_args += ['-J-Djvmci.Compiler=graal', '-X+T']
```

Comment:
Yeah, it isn't strictly needed in my experience, but since both the docs and Chris say I should use it, I did ;)